lib/Dialect/TritonGPU/Transforms/Pipeliner/SoftwarePipeliner.cpp

```diff
@@ -123,6 +127,9 @@ struct PipelinePass : public impl::TritonGPUPipelineBase<PipelinePass> {
       auto outerLoop = dyn_cast<scf::ForOp>(forOp->getParentOp());
       int loopNumStages = getNumStagesOrDefault(forOp);
       bool pipelined = pipelineLoop(forOp, loopNumStages);
+      if (DumpSWPFailure && !pipelined) {
+        forOp->emitRemark("SWP failes in inner most loop");
```

Comment:
@manman-ren was working on a framework to report performance warning. I don't think we should be adding command line option for every potential performance problem as this wouldn't scale.

Reply:
Yeah we don't need a separate flag for each warning. Warnings are guarded via MLIR_ENABLE_REMARK